uint32 id_req
---
uint32 id
float64 x
float64 y
float64 z
float64 angx
float64 angy
float64 angz
float64 speed
